src/Selenium2Library/locators/elementfinder.py

```diff
@@ -89,6 +90,10 @@ def _find_by_tag_name(self, browser, criteria, tag, constraints):
         return self._filter_elements(
             browser.find_elements_by_tag_name(criteria),
             tag, constraints)
+            
+    def _find_by_sc_locator(self, browser, criteria, tag, constraints):
+        js = "return isc.AutoTest.getElement('%s')" % criteria.replace("'", "\\'")
```

Comment:
A couple of questions. One, you use a double backslash "\\" when in all your references I see a double forward slash "//". For example, from [here](http://www.smartclient.com/smartgwtee-latest/javadoc/com/smartgwt/client/docs/UsingSelenium.html) I see

```
//ListGrid[ID='filterGrid']
 //DynamicForm[ID="autoTestForm"]/item[name=textField||title=textField||value=test||index=0||Class=TextItem]/element 
```

and from [codecentric](https://blog.codecentric.de/en/2010/12/testing-smartgwt-applications-with-selenium-and-robot-framework/)

```
//DynamicForm[ID="doSearchForm"]/item[name=mainRegisterBox]/element
//Button[ID="searchPerson"]/
//ListGrid[ID="hitList"].0.0
```

Second, you also are replacing within the given locator what seems to be a leading single quote. That makes me assume that you are expecting the user to write a locator like these

```
scLocator='ListGrid[ID='filterGrid']'
scLocator='//ListGrid[ID='filterGrid']'
```

Again I am not familiar with SmartClient and SmartGWT but the general usage for locators within Selenium2Library has been without the quotes. For example

```
Good
----
id=my element id
id=this-element

Not-used
--------
id='quoted-string'
id='no used locator'
```

I know myself I forget this and added quotes ever once in a while. If there is a possibility of having a double space in the locator I can see the need to have the sytax support quoted locators. But again in general that is not how they are used elsewhere.